### PR TITLE
SW-2612 Ungate Tracking-V1 feature in the web-app

### PR DESCRIPTION
**Pull request by karthikbtf**
- mark feature as enabled
- clean up tracking-v1 flags etc will be a followup PR (Jira SW-2356)
- added a change to show enabled features as disabled for selection in the opt-in page

### CODE REVIEW

**Comment by karthikbtf** [comment]
This won't be merged until we get approval to release tracking v1 to production.

**Review by constanzauanini — APPROVED** [review]
Looks good to merge when getting approval to release

**Pull request merged**
